app/helpers/application_helper.rb

```diff
@@ -155,6 +155,10 @@ def searchable?
     end
   end
 
+  def filter_columns?
+    controller_name == 'hosts' && controller.action_name == 'index'
```

Comment:
I guess it should be fine for now, since we only want to show the selector on hosts index page.

Reply:
I am not sure about this :/ I tried it without this check and it showed not only on hosts index page